packages/commonwealth/client/scripts/controllers/server/threads.ts

```diff
@@ -653,7 +653,7 @@ class ThreadsController {
   ): Promise<Thread[]> {
     const params = {
       chain: app.activeChainId(),
-      ids,
+      ids: ids.filter(Boolean),
```

Comment:
What does this line do?

Reply:
I was not able to reproduce the `NaN` error condition, so just hard code some `NaN` value from frontend to api. I believe there was some broken data that caused the original error. `ids.filter(Boolean)` just acts as a fail safe in case some data is broken/stale

Reply:
I mean specifically, what does passing in `Boolean` as a function do? Does it simply check truthy/falsy? Is there a more specific way we can confirm ids are valid (e.g. checking if it's a number or numeric string)?

Reply:
Oh it filter out JS falsy values so we are left with truthy values only. For a specific way, I think we can check if they are numbers (integers) but the data is used as it is from api so I dont think much in this context, but maybe it will in some other. Should we check if values are ints here as well?

Reply:
I think it's fine as is, just trying to understand. Resolved.